RELEASE.md

```diff
@@ -3,6 +3,9 @@ Add release notes for the upcoming release here.
 
 # Release 2.1.1:
 
+## Major features and improvements
+- Added optional pipeline argument to `get_data_from_kedro` (#72)
```

Comment:
Not major, but do you mind moving this up a bit? It should go under `# Upcoming release`. 2.1.1 was a previous release.